### PR TITLE
Fix Ant build script.

### LINKED ISSUE #7622: Ant is broken
Package `ant` is totally broken.

It existed before we even had `openjdk-17` package, but no idea if it even had any purpose or if anyone even tested it. I say we should look back into it now.

So basically
```
~  ant
/data/data/com.termux/files/usr/bin/ant: 41: /data/data/com.termux/files/usr/bin/ant: ANT_HOME: parameter not set
 ✘  ~  nvim $PREFIX/bin/ant
 ~  ant
java.lang.NullPointerException: Attempt to invoke virtual method 'java.io.File java.io.File.getParentFile()' on a null object reference
        at org.apache.tools.ant.launch.Launcher.run(Launcher.java:179)
        at org.apache.tools.ant.launch.Launcher.main(Launcher.java:101)
 ✘  ~ 
```

The first error is a dumb one, the script does `set -e -u`

The `-u` flag causes the program to error when an undefined shell variable is accessed, which makes no sense because those conditions check if a variable is not set in order to set it.

In the second line I invoke nvim to edit it and remove that `set -e -u` line and try again.

Now a Java error happens, no idea why but I'm gonna put the blame on dalvikvm, it's still running through dalvik, I say we should give it a try with the `openjdk-17` package now. This package will be much more useful with the openjdk package and a good alternative to gradle for some, so this should be re-packaged properly.

### PR DESCRIPTION
**Pull request by ravener**
Fixes #7622 

Not too experienced with the opt directory and stuff so not sure if it's the right place to put it but I just followed the Kotlin build script. It does have its own modules/jar files so it's very similar to kotlin's packaging I guess.

I nuked the `bin` folder because lot of the scripts is useless, `*.bat` and `*.cmd` scripts and some other misc scripts like python/perl ones. Only `ant` is all we need.

### CODE REVIEW

File: packages/ant/build.sh [hunk 1]
```diff
@@ -5,22 +5,16 @@ TERMUX_PKG_SRCURL=http://apache.mirrors.spacedump.net//ant/binaries/apache-ant-$
...
-
-	install $TERMUX_PKG_BUILDER_DIR/ant $TERMUX_PREFIX/bin/ant
-	perl -p -i -e "s%\@TERMUX_PREFIX\@%${TERMUX_PREFIX}%g" $TERMUX_PREFIX/bin/ant
+        mv ./bin/ant .
+	rm -f ./bin/*
+        mv ./ant ./bin/
+	rm -rf $TERMUX_PREFIX/opt/ant
+	mkdir -p $TERMUX_PREFIX/opt/ant
+	cp -r ./* $TERMUX_PREFIX/opt/ant
+        ln -sfr $TERMUX_PREFIX/opt/ant/bin/ant $PREFIX/bin/ant
```
Comment (xeffyr) [inline packages/ant/build.sh:19]: Inconsistent indentation and make sure you're using tabs.

File: packages/ant/build.sh [hunk 2]
```diff
@@ -5,22 +5,16 @@ TERMUX_PKG_SRCURL=http://apache.mirrors.spacedump.net//ant/binaries/apache-ant-$
 TERMUX_PKG_SHA256=ffc6c9eb2d153db76e18df70fa145744ddd0d754339df5a797a47603cfeaa8a6
```
Comment (xeffyr) [inline packages/ant/build.sh:5]: Package modifications require revision bump.

**Review by xeffyr — CHANGES_REQUESTED** [review]

File: packages/ant/build.sh (hunk 1 above)
Comment (ravener) [reply packages/ant/build.sh]: My bad, poorly edited from github editor in the browser. Just fixed it

File: packages/ant/build.sh (hunk 2 above)
Comment (ravener) [reply packages/ant/build.sh:6]: Whoops, that didn't come into mind.

**Review by xeffyr — APPROVED** [review]

**Pull request merged**